src/screens/DiscoveryScreenV2/elements/DiscoveryItemList.js

```diff
@@ -144,6 +145,19 @@ const DomainList = (props) => {
                 {item.description ? item.description : ''}
               </Text>
             )}
+            {item.comumnityInfo?.length > 0 && !item.user_id_follower && item.isUser && (
+              <View style={styles.communityTextContainer} testID="communityDesc">
+                <Text style={styles.textProfileFullName} numberOfLines={1} ellipsizeMode="tail">
+                  Also in{' '}
+                  <Text style={styles.communityText}>
+                    {`${item.comumnityInfo
+                      .slice(0, 3)
+                      .map((community) => `#${community}`)
+                      .join(', ')}`}
+                  </Text>
+                </Text>
+              </View>
+            )}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This new section for displaying community information seems to be only applicable when `item.isUser` is true and `item.user_id_follower` is false. Ensure that these conditions are always met when you expect this section to be displayed.



<!-- This is an auto-generated comment by OSS CodeRabbit -->